src/app/search/page.tsx

```diff
@@ -31,15 +31,15 @@ const StackedChart = dynamic(() => import('@/components/chart/StackedColumn').th
 });
 
 export default function Search() {
-  const [cityG1, setCityG1] = useState<string>(listaMunicipios[0]);
-  const [levelG1, setLevelG1] = useState<string>(optionsEtapas[0]);
-  const [cityG2, setCityG2] = useState<string>(listaMunicipios[0]);
-  const [levelG2, setLevelG2] = useState<string>(optionsEtapasGraf2[0]);
-  const [indicators, setIndicators] = useState<string>(indicadoresGraf2[0]);
-  const [rede, setRede] = useState<string>(redeGraf2[0]);
-  const [rankYear, setRankYear] = useState<string>(anos[0]);
-  const [levelRank, setLevelRank] = useState<string>(optionsEtapas[0]);
-  const [rankOrder, setRankOrder] = useState(rank[0]);
+  const [cityG1, setCityG1] = useState<string>(listaMunicipios[0].value);
```

Comment:
Para as cidades, usaremos number em vez de string, logo, fazer conversão de string para number onde for setar cidades. A conversão deve ser feita apenas na page Search, para que o componente filter continue da forma que está, aceitando um array com value: string, já que os demais são strings.